src/web/app/pipes/pipes.module.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { Pipes } from './pipes.module';
+import { ResponseStatusPipe } from './sessionResponseStatus.pipe';
+import { SubmissionStatusPipe } from './sessionSubmissionStatus.pipe';
+
+describe('Pipes', () => {
+  let pipes: Pipes;
+  let responseStatusPipe: ResponseStatusPipe;
+  let submissionStatusPipe: SubmissionStatusPipe;
+
+  beforeEach(() => {
+    pipes = new Pipes;
+    responseStatusPipe = new ResponseStatusPipe();
+    submissionStatusPipe = new SubmissionStatusPipe();
+  });
+
+  /**
+   * Tests for response status pipe.
```

Comment:
While this works, this is not ideal because you're not really testing the Pipes module per se. What you should do instead is testing the pipes themselves with `session-response-status.pipe.spec.ts`.